MovieApp/MovieApp/NewFeedsItem.h

```diff
@@ -0,0 +1,20 @@
+//
+//  NewFeedsItem.h
+//  MovieApp
+//
+//  Created by user on 22/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NewFeedsItem : NSObject
+
+@property (strong, nonatomic) NSString *headline;
+@property (strong, nonatomic) NSString *text;
+@property (strong, nonatomic) NSString *webPage;
+//@property (strong, nonatomic) NSDate *dateOfArticle;
+
+- (id)initWithHeadline:(NSString *)pheadline andWithText:(NSString *)ptext andWithWebPage:(NSString *)pwebPage;
```

Comment:
Too much unnecessary repetition in a name of the method as well as unclear argument names: `initWithHeadline:andWithText:andWithWebPage:`. You can rename it to something like: `initWithHeadline:text:detailsUrlPath:`.